src/lib/common/sem.cpp

```diff
@@ -348,6 +350,8 @@ int get_curl_context(const std::string& key, struct curl_context* pcc)
 
       if (pm == NULL)
       {
+        pthread_mutex_unlock(&contexts_mutex);
+        LM_M(("KZ: released contexts_mutex"));
         LM_E(("malloc"));
```

Comment:
By the way, please ensure that all the LM_E touched in this PR take into accoun the "convention" for texts:: https://github.com/telefonicaid/fiware-orion/blob/develop/doc/manuals/admin/logs.md#alarm-conditions, e.g. probably they should use "Runtime Error (...)".